python_modules/dagster/dagster/_core/definitions/asset_condition_evaluation_context.py

```diff
@@ -162,6 +163,18 @@ def previous_true_subset(self) -> AssetSubset:
             return self.empty_subset()
         return self.previous_evaluation.true_subset
 
+    @property
```

Comment:
I don't have all the context here, but it is a bit of a confusing state to have to know when to use a property prefixed with `previous_` directly on this class versus a subproperty of `previous_evaluation` or `previous_evaluation_state`. Potential source of bugs

Reply:
yeah reasonable point -- the main motivation for this pattern was to avoid having to do a double `None` check of `some_var = (context.previous_evaluation.var or <default val>) if context.previous_evaluation is not None else <default val>

Generally there's no problem with accessing the value in the more roundabout way, it'll just result in you having to handle more cases to get the type system happy with you.